apps/src/applab/designElements/dropdown.jsx

```diff
@@ -88,6 +88,16 @@ module.exports = {
     element.style.height = '30px';
     element.style.fontSize = '14px';
     element.style.margin = '0';
+    element.style.color = '#fff';
+    element.style.backgroundColor = '#1abc9c';
```

Comment:
In this case, not necessarily because our design mode code looks at the color attribute (Rather than a class attribute) to populate the property table.

I guess one option would be to do something like call getComputedStyle (I think there's some function like this) instead of just grabbing the attribute.